content/en/docs/reference/scheduling/policies.md

```diff
@@ -16,6 +16,10 @@ You can set a scheduling policy by running
 `kube-scheduler --policy-configmap <ConfigMap>`
 and using the [Policy type](/docs/reference/config-api/kube-scheduler-policy-config.v1/).
 
+{{< warning >}}
+This scheduling policy is removed in v1beta3, associated flags `policy-config-file`, `policy-configmap`, `policy-configmap-namespace` and `use-legacy-policy-config` are also removed. Instead, use the [Scheduler Configuration](/docs/reference/scheduling/config/) to achieve similar behavior.
```

Comment:
this note is not about v1beta3. It's just removed altogether.

I think we should remove the rest of the file.

Reply:
Do you mean we just remove the file about scheduling policy all together, what about the users still using lower version like v1beta1, they may want to refer to this article for details. 

Just to confirm whether I understood your words correctly.

Reply:
first of all, the scheduling policy is not tied to the component config API. They are alternatives to each other.

In 1.23, we removed the scheduling policy entirely. If somebody wants to use it, they have to run k8s 1.22 or older. The website documentation always refers to the latest version, then we should remove the contents of the file and just leave the note.

Users can always find documentation for older releases like this: https://v1-21.docs.kubernetes.io/docs/home/

Reply:
thanks for explanation, since scheduling policy is removed entirely, I'll delete this file then.

Reply:
I would leave the note and remove the rest of the file.

Reply:
preview [here](https://deploy-preview-30404--kubernetes-io-vnext-staging.netlify.app/docs/reference/scheduling/policies/)